contrib/cirrus/runner.sh

```diff
@@ -233,6 +233,25 @@ function _run_consistency() {
     SUGGESTION="run 'make generate-bindings' and commit all changes" ./hack/tree_status.sh
     make completions
     SUGGESTION="run 'make completions' and commit all changes" ./hack/tree_status.sh
+
+    if  [[ -z "$CIRRUS_TAG" ]] && \
+        req_env_vars CIRRUS_CHANGE_IN_REPO CIRRUS_PR DEST_BRANCH
+    then
+        local base diffs regex i
+        # Prevent this check from detecting itself (0x69 == "i")
+        i=$(echo -ne '\x69')
```

Comment:
So....... you could just say `i=i` :wink:

Reply:
oh durrr, ya, I s'pose I could.  Thanks, I'll fix it.